src/javascript/track.js

```diff
@@ -56,14 +57,14 @@ export default class DropTrack {
                 this.isUserHoldingNote = false;
                 clearInterval(countInterval);
               }
-            }, 100);
+            }, 100 / this.vm.playbackSpeed);
```

Comment:
I hope `vm.playbackSpeed` won't ever be `0` or coerce to `0` or `NaN`

Reply:
I hope the same, tho it shouldn't